packages/shared/src/types/targeting/shared.ts

```diff
@@ -33,3 +34,12 @@ export interface PurchaseInfo {
     userType: purchaseInfoUser;
     product: purchaseInfoProduct;
 }
+
+export const abandonedBasketSchema = z.object({
+    amount: z.union([z.number(), z.literal('other')]),
+    billingPeriod: z.string(),
```

Comment:
how about making these other types more strict?

Reply:
ie match literals like `monthly` ?

Reply:
yes ideally matching the types here https://github.com/guardian/support-frontend/pull/5972/files#diff-f187ba8d13a2ad5cfda37a4e6a3b77f33e476e50f0175d26d0b24d5e25a965b9R8
I can see this being particularly useful if we need to target different messages based on e.g. the product type